src/tracing/mod.rs

```diff
@@ -32,6 +35,11 @@ impl<'opts> TracingOpts<'opts> {
             filter_directives: &opts.logging.tracing_filter,
             trace_spans: &opts.logging.trace_spans,
             json_log_path: opts.logging.log_json.as_deref(),
+            output_stream: if opts.tui {
+                OutputStream::Stderr
+            } else {
+                OutputStream::Stdout
+            },
```

Comment:
`tracing-human-layer` uses `OutputStream::Stdout` by default, so this is behavior preserving when you're not in TUI mode.